prov/efa/src/rxr/rxr_pkt_entry.h

```diff
@@ -114,6 +114,17 @@ struct rxr_pkt_sendv {
  * is not registered (when it is unexpected or out-of-order). A new packet entry will be cloned
  * using endpoint's read_copy_pkt_pool, whose memory was registered.
  */
+
+struct efa_send_wr {
+	struct ibv_send_wr wr;
+	struct ibv_sge sge[2]; // EFA device supports a maximum of 2 iov/SGE
```

Comment:
"//" should be "/**< ... */" for doxygen comment

Reply:
done